images/cert-manager-acmesolver/main.tf

```diff
@@ -0,0 +1,76 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+variable "target_repository" {
+  description = "The docker repo into which the image and attestations should be published."
+}
+
+module "latest" {
+  source = "../../tflib/publisher"
+
+  target_repository = var.target_repository
+  config            = file("${path.module}/configs/latest.apko.yaml")
+}
+
+module "dev" { source = "../../tflib/dev-subvariant" }
+
+module "latest-dev" {
+  source = "../../tflib/publisher"
+
+  target_repository = var.target_repository
+  # Make the dev variant an explicit extension of the
+  # locked original.
+  config         = jsonencode(module.latest.config)
+  extra_packages = module.dev.extra_packages
+}
+
+module "latest-cm-dev" {
+  source = "../../tflib/publisher"
+
+  target_repository = var.target_repository
+
+  config = jsonencode(module.latest.config)
+  extra_packages = concat(module.dev.extra_packages, [
+    "cmctl",
+  ])
+}
+
+module "version-tags" {
+  source  = "../../tflib/version-tags"
+  package = "cert-manager-acmesolver"
+  config  = module.latest.config
+}
+
+module "test-latest" {
+  source = "./tests"
+  digest = module.latest.image_ref
+}
+
+module "test-latest-dev" {
+  source = "./tests"
+  digest = module.latest-dev.image_ref
+}
+
+module "test-latest-cm-dev" {
+  source = "./tests"
+  digest = module.latest-cm-dev.image_ref
+}
+
+module "tagger" {
+  source = "../../tflib/tagger"
+
+  depends_on = [
+    module.test-latest,
+    module.test-latest-dev,
+    module.test-latest-cm-dev,
+  ]
+
+  tags = merge(
+    { for t in toset(concat(["latest"], module.version-tags.tag_list)) : t => module.latest.image_ref },
+    { for t in toset(concat(["latest"], module.version-tags.tag_list)) : "${t}-dev" => module.latest-dev.image_ref },
+    { for t in toset(concat(["latest"], module.version-tags.tag_list)) : "${t}-cm-dev" => module.latest-cm-dev.image_ref },
```

Comment:
The current semantics puts `cmctl` into the normal `-dev` there isn't another tag